bip32/example_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2014 The btcsuite developers
```

Comment:
I'm nearly certain we've been deliberately preserving these comments as an acknowledgement to the original codebase

Reply:
Yeah if you do a `grep -rsn --include=*go Copyright`, you'll see a few different copyrights and development teams mentioned

Reply:
ah ok